### PR TITLE
Force interactive mode for update command

### PR DESCRIPTION
**Pull request by lwronski**
The IMO the `update` command should enable interactive mode by default.

### CODE REVIEW

**Review by Gedochao — APPROVED** [review]
Makes sense.

File: modules/cli/src/main/scala/scala/cli/commands/util/VerbosityOptionsUtil.scala [hunk 1]
```diff
@@ -5,6 +5,7 @@ import scala.build.interactive.Interactive._
 
 object VerbosityOptionsUtil {
   implicit class VerbosityOptionsOps(v: VerbosityOptions) {
-    val interactiveInstance = if (v.interactive.getOrElse(false)) InteractiveAsk else InteractiveNop
+    def interactiveInstance(forceEnable: Boolean = false) =
+      if (v.interactive.getOrElse(false) || forceEnable) InteractiveAsk else InteractiveNop
```
Comment (alexarchambault) [inline modules/cli/src/main/scala/scala/cli/commands/util/VerbosityOptionsUtil.scala]: Why not make the parameter the default value?
```suggestion
      if (v.interactive.getOrElse(forceEnable)) InteractiveAsk else InteractiveNop
```

Comment (alexarchambault) [reply modules/cli/src/main/scala/scala/cli/commands/util/VerbosityOptionsUtil.scala]: If forced, the command-line option is simply ignored, while this way, users can override the default.

Comment (lwronski) [reply modules/cli/src/main/scala/scala/cli/commands/util/VerbosityOptionsUtil.scala]: Makes sense, I applied your suggestion

**Pull request merged**